### ISSUE TITLE
이미지 사이즈

### ISSUE THREAD

**Issue by kwanlae** [plusfriend/auto_reply#398]
`"photo": {
  "url": "https://photo.src",
    "width": 640,
    "height": 480
}`

로 이미지를 걸때 아이폰과 안드로이드폰에서 다른 사이즈로 표시되는 것 같습니다. 이것 외에도 iOS 와 안드로이드간에 이미지가 다르게 표시되는 경우가 있었는데, 확인해 주실 수 있으신가요?

혹은 이미지외에 카톡창에 표현되는 모든 경우에 있어서 디바이스나 OS 의 특성을 타는 다른 경우가 또 있는지요.

**Comment by kwanlae** [comment]
몇가지 케이스 테스트를 해 본 결과 iOS 는 원본 이미지 사이즈와 무관하게 width 와 height 로 조절할 수 있는 반면, 안드로이드는 그런 것 같지 않고, width x height 값으로 줄여진 이미지를 다시 대화창 사이즈에 맞게 확대(?) 하는 것 같습니다.

조금 크리티컬한 이슈라 빠른 확인 부탁드리겠습니다.

**Comment by ilhanjo** [comment]
이미지 자체의 크기는 변함없지만 채팅창에 올라오는 미리보기 사진의 경우인가요?

**Comment by kwanlae** [comment]
@ilhanjo 네, 카카오톡 채팅창에 올라오는 이미지 사이즈가 디바이스(테스트 해본 결과로는 iOS / Android)에 따라 달리 표현되고, 안드로이드에서는 width, height 값이 전혀 적용되지 않고 있는 듯 합니다.

**Comment by khhwang838** [comment]
적용되지 않는다는 말씀은 이미지 원본 사이즈에 따라 카톡에서 자동으로 확대/축소를 해버린다는 말씀이신가요?
